src/transformers/models/llava_next/modeling_llava_next.py

```diff
@@ -509,6 +510,12 @@ def _merge_input_ids_with_image_features(
         image_token_index = image_token_index if image_token_index is not None else self.config.image_token_index
         ignore_index = ignore_index if ignore_index is not None else self.config.ignore_index
 
+        if self.training and self.padding_side == "left":
+            warnings.warn("padding is left even though in training mode. If that's intended, ignore this warning")
+
```

Comment:
```suggestion
        if self.training and self.padding_side == "left":
            logger.warning_once(
                "Padding side is set to 'left' but the model is in training mode. For training "
                "it is recommended to set `model.padding_side='right' and `processor.tokenizer.padding_side='right'`. "
                "If that's intended, ignore this warning"
            )
```

nit, we can be more detailed in warning message, Sorry, my bad for that example message you copied